### PR TITLE
Collection Cards Magic The Gathering!

### PR DESCRIPTION
**Pull request by BregornOriginal**
## Home page :100:

✅ When the page loads, the web app loads the list of cards from the API selected.

✅ Along with the category name, there is displayed the Converted Mana Cost to convert a Card's Mana.

✅ When the user clicks (or taps) on a category item, the application navigates to the details page.

## Detail Page :100:

✅ On the details page, the web app retrieves data from the API to show detailed data in the category. 

✅ When the user clicks on the "Back" button (<) or on the logo icon, the user navigates to the home page.

- Add tests for different components
- Add integration test
- Use a grid container for the list of cards
- Add a hover effect for the cards
- Add a custom font for the appearance of Magic Gatherings Cards
- Also were added styles for the desktop version.

### Technologies

🔷 React-Redux

🔷 ES6

🔷 CSS

🔷 Jest

### Tools

💠VISUAL STUDIO CODE

💠GITHUB-DESKTOP

💠GITHUB-ACTIONS

💠LINTERS

💠LOOM

## LOOM Video

[VIDEO](https://www.loom.com/share/3fcc9d0dd634403cb416dbdd29667dd0)

## Live Demo

[Magic The Gathering](https://bregorn-magic-the-gathering.netlify.app/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,111 @@
...
+## Authors
+
+👤 Julio Miguel Gagliardi
+
+- GitHub: [@BregornOriginal](https://github.com/BregornOriginal)
+- Twitter: [@Bregorn](https://twitter.com/home)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/julio-gagliardi/)
+
+> I want to say thank you to Nelson Sakwa in Behance for share her design
+
```
Comment (Mihndim2020) [inline README.md:102]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)(checkout the `Design section` of the project requirements) for this project and best practice, " `The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` ", kindly add an `Acknowledgements` section to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @BregornOriginal, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Descriptive `PR` summary,
:heavy_check_mark:  Great job with UI,
:heavy_check_mark:  Clean/Organized code,
:heavy_check_mark:  Great job with video presentation,
:heavy_check_mark:  No Linter errors,
:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (BregornOriginal) [reply README.md:104]: Ok, thank you for the review! Added the new section of Acknowledgements in the readme!

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @BregornOriginal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**